src/internal/_meta.js

```diff
@@ -0,0 +1,32 @@
+module.exports = (function() {
+    // Should we use a Symbol or let users have free access to metadata
+    // which could be useful for extensions.
```

Comment:
We can always use a symbol and later decide to publicly expose the symbol if we choose.